examples/match_statements/src/main.sw

```diff
@@ -36,6 +42,23 @@ fn main() -> u64 {
         },
     };
 
+    // Match an enum
+    let e = SomeEnum::A(42);
+    let v = match e {
+        SomeEnum::A(val) => {
```

Comment:
Add a TODO here that the `{ }` aren't needed, but are only there because of <link to PR> a bug in the formatter.